tests/unit/code_provider/test_git_safe.py

```
@@ -0,0 +1,574 @@
+
+        assert result == "result"
+
+
+class TestInstallSigsegvHandler:
+    """Tests for install_sigsegv_handler function."""
+
+    def test_install_sigsegv_handler_does_not_crash(self):
+        """install_sigsegv_handler should not raise any exception."""
+        try:
+            install_sigsegv_handler()
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)